### DISCUSSION TITLE
YouTube summarization

### DISCUSSION THREAD

**Discussion by i486** [adamlui/ai-web-extensions#44] [Ideas]
Hey @adamlui 

I've been a big fan of your ChatGPT related apps and userscripts – they've been incredibly helpful.

I wanted to bring up a topic that's been on my mind for last few months. The existing YouTube summarization add-ons are either no longer functional or have been abandoned, and some unfortunate cases involve ad agencies making things very worse.

Given your track record for creating reliable tools for ChatGPT, I thought you might be the perfect person to tackle this issue. 
Would you consider developing a YouTube summarization add-on or userscript? The community could greatly benefit from a tool that is not only effective but also free from the pitfalls we've seen in all other options.

I understand your time is valuable, and any consideration you give to this would be highly appreciated by the community.

Looking forward to hearing your thoughts.

Best regards,
John

**Comment by adamlui** [comment]
Thanks @i486, I will work on this sometime soon for you
